packages/client/src/__tests__/integration/happy/client-engine/test.ts

```diff
@@ -92,7 +92,7 @@ function buildTests() {
             ]
             `)
 
-          expect(prisma._clientEngineType).toMatch(expectedClientEngine)
```

Comment:
This property was removed, as it was really only used for this test. Now using an equivalent based on the generator config.